patroni/common_config.py

```diff
@@ -0,0 +1,9 @@
+def is_standby_cluster(config):
+    """ Check whether or not provided configuration describes a standby cluster.
+        Config can be both patroni config or cluster.config.data
+    """
+    return config is not None and (
```

Comment:
```python
isinstance(config, dict) and (config.get('host') or config.get('port') or config.get('restore_command'))
```

Because:
```python
In [1]: '' is not None
Out[1]: True
```